src/BuildScriptGenerator/Golang/GolangBashBuildSnippet.sh.tpl

```diff
@@ -0,0 +1,22 @@
+﻿# This will build go build applications
+# Requirements:
+#	./go.mod
+#	./main.go
+
+echo "   "
+echo "Using Go version: "
+go version
+echo "   " 
+echo "   "
+
+# TODO: add time metrics
```

Comment:
you should still add time metrics because we do have telemetry on that

Reply:
Agreed, added

Reply:
@arroyc I noticed that golang build app takes less than 1 second to build. So output shows: `Golang app built in 0 sec(s).` I am consider in removing timing metric for glang build. What are your thoughts?